src/Pages/GlobalConfigurations/Authorization/types.ts

```diff
@@ -74,7 +74,8 @@ export interface UserGroupDTO {
     usersCount?: number
 }
 
-export interface UserGroupType extends Required<Pick<UserGroupDTO, 'description' | 'name' | 'usersCount'>> {
+export interface UserGroupType
+    extends Required<Pick<UserGroupDTO, 'description' | 'name' | 'usersCount' | 'identifier'>> {
```

Comment:
Need identifier since need to create a map of identifier to name in user group in approval modal since have to show name

Reply:
Since in other configs i am receiving a key named identifier itself, ideally userGroupId should be renamed